src/index.spec.js

```diff
@@ -621,37 +633,49 @@ describe('Dropzone', () => {
       expect(child).toHaveProp('isDragReject', false)
     })
 
-    it('should add valid files to rejected files on a multple drop when multiple false', () => {
+    it('should add valid files to rejected files on a multple drop when multiple false', done => {
       const dropzone = mount(<Dropzone accept="image/*" onDrop={dropSpy} multiple={false} />)
       dropzone.simulate('drop', { dataTransfer: { files: images } })
-      const rejected = dropSpy.firstCall.args[0]
-      expect(rejected.length).toEqual(1)
+      setTimeout(() => {
```

Comment:
I’d prefer us using async await here instead of timeouts.

Reply:
Or anything that’s fits better from https://facebook.github.io/jest/docs/en/asynchronous.html#promises

Reply:
I agree with that @okonet. We should be looking to move any promises to async await where we can.

Reply:
@okonet @rkmarks I prefer async/await as well, but in this case I think there is nothing to 'await' on. I will explore how I can do this, please suggest if you can think of a way